python/maths/sum_of_all_elements_till_nth_row_pascals_triangle.py

```diff
@@ -0,0 +1,22 @@
+# Input  : 2
+# Output : 7 
```

Comment:
I tested the code with the sample input provided. I am getting the output as 3 instead of 7 as documented by you. Please contact me on [Slack](https://join.slack.com/t/geekquad/shared_invite/zt-jtdqxt2r-i9R7vvNPzveImCBBM~eahw).